app/scripts/xprizeDemo.xml

```diff
@@ -0,0 +1,217 @@
+<!-- Copyright (C) 2021 Fondazione Istituto Italiano di Tecnologia (IIT) -->
+<!-- All Rights Reserved. -->
+<!-- Author: Giulio Romualdi <giulio.romualdi@iit.it> -->
+
+<application>
+
+  <name>Xrprize Demo (Virtualizer Xsens SenseGlove)</name>
+  <description>Application to run the ANA Avatar XPRIZE demo.</description>
+  <version>1.0</version>
+  <authors>
+    <author email="giulio.romualdi@iit.it">Giulio Romualdi</author>
+  </authors>
+
+  <!-- Modules -->
+  <!-- icub-head -->
+
+  <module>
+    <name>yarpdev</name>
+    <parameters> --from camera/ServerGrabberDualDragon.ini --split true</parameters>
+    <node>icub-head</node>
+  </module>
+
```

Comment:
Add the module related to the transform server.

```
<module>
    <name>yarpdev</name>
    <parameters> --device transformServer --ROS::enable_publisher 1 --ROS::enable_subscriber 0</parameters>
    <node>icub-console-gui</node>
  </module>
```

Reply:
See https://github.com/robotology/walking-teleoperation/pull/47#issuecomment-767554588